novawallet/Modules/Vote/Governance/TinderGov/TinderGovViewFactory.swift

```diff
@@ -2,17 +2,57 @@ import Foundation
 import SoraFoundation
 
 struct TinderGovViewFactory {
-    static func createView(with referendums: [ReferendumLocal]) -> TinderGovViewProtocol? {
+    static func createView(
+        observableState: Observable<NotEqualWrapper<[ReferendumIdLocal: ReferendumLocal]>>,
+        sharedState: GovernanceSharedState
```

Comment:
As we agreed let's put observable state to the shared state and clean on account/chain change

Reply:
Will present in the coming PR